tslint.json

```diff
@@ -106,9 +106,6 @@
         "use-life-cycle-interface": true,
         "use-pipe-transform-interface": false,
         "component-class-suffix": true,
-        "directive-class-suffix": true,
-        "no-access-missing-member": true,
-        "templates-use-public": true,
-        "invoke-injectable": true
```

Comment:
are the rules removed or are they giving lint issues?

Reply:
yeah